### PR TITLE
Cleanup dependencies

### PR DESCRIPTION
**Pull request by dorschw**
related: https://jira-dc.paloaltonetworks.com/browse/CIAC-9541
Infra-side MR: https://gitlab.xdr.pan.local/xdr/cortex-content/infra/-/merge_requests/369

### CODE REVIEW

File: .hooks/bootstrap [hunk 1]
```diff
@@ -89,23 +92,13 @@ else
 fi
 
 if [ -n "${CI}" ]; then
-    retry_count=3
-    sleep_time=10
-    echo "Detected CI environment. Running extra requirements"
-    echo "Check if poetry files are valid"
+    echo "Detected CI environment"
+    echo "Checking whether poetry files are valid" # TODO do we need that, when we have pre-commit?
```
Comment (kobymeir) [inline .hooks/bootstrap:96]: remove the comment
```suggestion
    echo "Checking whether poetry files are valid"
```

File: pyproject.toml [hunk 2]
```diff
@@ -12,83 +12,26 @@ url = "https://pypi.org/simple/"
...
+pytest = "^7.4.4"
+requests = "^2.32.3"
+urllib3 = "^2.2.2"
 blessings = "^1.7"
-coloredlogs = "^15.0.1"
-pandas = "^1.3.3"
-prettytable = "^3.2.0"
-pyspellchecker = "^0.6.2"
-sendgrid = "^6.6.0"
+PyGithub = "^1.59"    # TODO v2?
```
Comment (kobymeir) [inline pyproject.toml:22]:
```suggestion
PyGithub = "^1.59"
```
talk to @kgal-pan about this

File: pyproject.toml [hunk 3]
```diff
@@ -12,83 +12,26 @@ url = "https://pypi.org/simple/"
...
-Faker = "^15.3.4"
+
+[tool.poetry.group.dev.dependencies]
+demisto-sdk = "1.31.5"
+pytest = ">=7.1.2"
+requests-mock = ">=1.9.3"
+pytest-mock = ">=3.7.0"
+freezegun = ">=1.1.0"
+dateparser = ">=1.2.0"
+ruff = "*"
```
Comment (kobymeir) [inline pyproject.toml:34]: why it's unlocked here?

File: Packs/CommonScripts/Scripts/DBotUpdateLogoURLPhishing/DBotUpdateLogoURLPhishing.py [hunk 4]
```diff
@@ -159,7 +159,7 @@ def get_concat_logo_single_image(logo_list):
     y_offset = 0
     new_size = (width_new, height_new)
     for im in images:
-        im = im.resize(new_size)
+        im = im.resize(new_size)  # type:ignore[assignment]
```
Comment (kobymeir) [inline Packs/CommonScripts/Scripts/DBotUpdateLogoURLPhishing/DBotUpdateLogoURLPhishing.py:162]: why in this PR and not split it?

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 3 above)
Comment (dorschw) [reply pyproject.toml:34]: so we don't have to maintain it really, pre-commit enforces the same version for everyone anyway

File: pyproject.toml [hunk 5]
```diff
@@ -12,83 +12,26 @@ url = "https://pypi.org/simple/"
...
```
Comment (kobymeir) [inline pyproject.toml:34]:
```suggestion
ruff = "*"  # See `.pre-commit-config-template` for version used in pre-commit. Version here doesn't matter.
```
?

**Review by kobymeir — APPROVED** [review]

File: .hooks/bootstrap [hunk 6]
```diff
@@ -89,26 +89,16 @@ else
...
-            sleep "${sleep_time}"
-        fi
-    done
-    exit_on_error "${exit_code}" "Failed to install dependencies after ${retry_count} tries"
+    poetry install --no-interaction
+    exit_on_error "${exit_code}" "Failed to install dependencies"
 else
     echo "Detected local environment"
-    echo "Check if poetry files are valid"
+    echo "Check if poetry files are valid" # TODO do we need that, when we have pre-commit?
```
Comment (dorschw) [inline .hooks/bootstrap:101]:
```suggestion
    echo "Check if poetry files are valid"
```

**Pull request merged**